<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>multChrome</class>
 <widget class="QMainWindow" name="multChrome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chrome 多开管理器</string>
  </property>
  <property name="windowIcon">
   <iconset theme="tmp.ico"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,1,2,14">
      <item>
       <widget class="QLabel" name="title_text">
        <property name="font">
         <font>
          <family>思源黑体 CN Medium</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Chrome 多开管理器</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_0" stretch="7,3">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>添加用户</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="7,3">
           <item>
            <widget class="QLineEdit" name="add_user_entry">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>用户名</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="add_user_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>添加</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="about_grp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>关于</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QLabel" name="about_text">
             <property name="text">
              <string>v1.3.0 @Lokkon</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_1" stretch="7,3">
        <item>
         <widget class="QGroupBox" name="user_list_grp">
          <property name="title">
           <string>用户列表</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QListWidget" name="user_lb">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="movement">
              <enum>QListView::Static</enum>
             </property>
             <property name="resizeMode">
              <enum>QListView::Fixed</enum>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="function_layout" stretch="4,3,2">
          <item>
           <widget class="QGroupBox" name="basic_function_grp">
            <property name="title">
             <string>基本功能</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QPushButton" name="open_button">
               <property name="text">
                <string>打开</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="remove_button">
               <property name="text">
                <string>移除</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="open_folder_button">
               <property name="text">
                <string>打开用户文件夹</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="open_url_grp">
            <property name="title">
             <string>打开指定网站</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLineEdit" name="open_url_entry">
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>输入url</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="open_url_button">
               <property name="text">
                <string>打开</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="setting_grp">
            <property name="title">
             <string>设置</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QPushButton" name="chrome_path_button">
               <property name="text">
                <string>Chrome路径</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
